azurelinuxagent/common/cgroupconfigurator.py

```diff
@@ -409,8 +409,9 @@ def __set_cpu_quota(quota):
             NOTE: This is done using a dropin file in the default dropin directory; any local overrides on the VM will take precedence
             over this setting.
             """
-            logger.info("Setting agent's CPUQuota to {0}%", quota)
-            if CGroupConfigurator._Impl.__try_set_cpu_quota(quota):
+            quota_percentage = "{0}%".format(quota)
+            logger.info("Setting agent's CPUQuota to {0}", quota_percentage)
```

Comment:
`_log_cgroup_info` instead maybe (for the Kusto log)?

Reply:
sounds good; i also did _log_cgroup_warning when we fail to change the quota